README.md

```diff
@@ -1,5 +1,6 @@
 # Go SDK for Serverless Workflow
 
+To get started with the project standards, please take a look in the [contributors_guide](CONTRIBUTORS_GUIDE.md).  
```

Comment:
I think we can have a `Contribution` section in this file instead, wdyt?

Reply:
works as well, one file less :)